src/main/java/uk/gov/ons/ctp/integration/contactcentresvc/service/impl/CaseServiceImpl.java

```diff
@@ -531,9 +531,8 @@ public ResponseDTO reportRefusal(UUID caseId, RefusalRequestDTO requestBodyDTO)
         .debug("Processing refusal for case with reported dateTime");
 
     // Create and publish a respondent refusal event
-    UUID refusalCaseId = caseId == null ? new UUID(0, 0) : caseId;
     RespondentRefusalDetails refusalPayload =
-        createRespondentRefusalPayload(refusalCaseId, requestBodyDTO);
+        createRespondentRefusalPayload(caseId, requestBodyDTO);
```

Comment:
The code coverage tool gives me the message "1 of 2 branches missed" about line 542 below.

Reply:
Good spot. 
Remaining support for unknown caseId now removed.